src/agent/onefuzz-task/src/tasks/fuzz/supervisor.rs

```diff
@@ -252,57 +290,19 @@ async fn start_supervisor(
         None
     };
 
-    let expand = Expand::new(&config.common.machine_identity)
-        .machine_id()
-        .supervisor_exe(&config.supervisor_exe)
-        .supervisor_options(&config.supervisor_options)
+    let expand = config
+        .get_expand()?
         .runtime_dir(&runtime_dir)
         .crashes(&crashes.local_path)
+        .input_corpus(&inputs.local_path) // Why isn't this value in the config? It's not super clear to me from looking at the calling code.
```

Comment:
You could create an issue to look into that after this PR merges. I have a similar gut feeling that we _shouldn't_ need it and it would clean up the code by removing it.

Reply:
If you create the issue, make sure to include all of the locations in the code that we know about that have this potential issue.

Reply:
https://github.com/microsoft/onefuzz/issues/3565